components/bl-leaflet-map/src/helpers/map.js

```diff
@@ -86,21 +86,61 @@ export function initMap(component, eventHandlers, map, currentLayer, uid) {
   });
 }
 
+function validateCircle(circle) {
+  if (!circle.point) {
+    console.error('Circle error!\n Circle should store "point" property with latlng object in\n', circle);
+
+    return false;
+  }
+
+  const { point: { lat, lng }, radius, description } = circle;
+  const isAllPropertiesAreNumbers = [lat, lng, radius].every(value => {
+    const result = !isNaN(value);
+
+    if (!result) {
+      console.error(`Circle error!\n Expected number but received "${ value }" in\n`, circle);
+    }
+
+    return result;
+  });
+
+  if (!isAllPropertiesAreNumbers) {
+    return false;
+  }
+
+  if (description !== undefined && typeof description !== 'string') {
+    console.error(`Circle Erorr!\n Expected description type string but received "${ description }" in\n`, circle);
```

Comment:
Expected - <type>, but received - <value>. Maybe we need change it to "receiver <type>"?